log-collector/src/main/scala/io/phdata/pulse/logcollector/LogCollector.scala

```diff
@@ -46,31 +48,52 @@ object LogCollector extends LazyLogging {
     val solrServer  = new CloudSolrServer(cliParser.zkHosts())
     val solrService = new SolrService(cliParser.zkHosts(), solrServer)
 
+    val routes = new LogCollectorRoutes(solrService)
+
+    cliParser.mode() match {
+      case "kafka" => {
+        consume(solrService, cliParser.kafkaProps(), cliParser.topic())
```

Comment:
nit: I think 'kafka' and 'http' might be better names for these methods. You could say the http server is also consuming.

Reply:
Fixed